website/docs/reference/node-selection/node-selection.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Node selection"
+description: "Learn how to select and exclude resources in dbt."
+sidebar_label: "About node selection"
+---
+
+This page outlines the steps dbt follows when selecting resources. To understand dbt resource selection:
```

Comment:
i'd love to clarify if we're talking about nodes here or resources?

Reply:
if we're talking commands and flags, it seems like we mean resources adn node nodes?